README.md

```diff
@@ -70,12 +72,13 @@ They are not suitable for most production use-cases because they keep all data i
    `ferretdb` runs FerretDB.
 
 2. Start services with `docker compose up -d`.
+   To be sure that you've fetched the latest version you can precede the command with `docker compose pull`.
 
 3. If you have `mongosh` installed, just run it to connect to FerretDB.
    If not, run the following command to run `mongosh` inside the temporary MongoDB container, attaching to the same Docker network:
 
    ```sh
-   docker run --rm -it --network=ferretdb --entrypoint=mongosh mongo mongodb://ferretdb/
+   docker run --rm -it --network=ferretdb --entrypoint=mongosh mongo mongodb://ferretdb/ -u username -p password
```

Comment:
I ran setup step by step until here I get `MongoServerError: Unsupported mechanism 'PLAIN'` error. Does it work for you?

Reply:
Nice catch! 

I was sure that I've connected with success, but probably I've changed too many things and didn't track the changes. 

Adding the `--authenticationMechanism=PLAIN` flag fix the issue. To ensure that everything works correctly I've run everything on seperate environment :smile: